### ISSUE TITLE
eslint-import-resolver-node and typescript error

### ISSUE THREAD

**Issue by yshrsmz** [import-js/eslint-plugin-import#2185]
Since eslint-import-resolver-node v0.3.5, following line starts showing error

```typescript
import { GetterTree, ActionTree, MutationTree } from 'vuex'
```

```
  1:10  warning  GetterTree not found in 'vuex'    import/named
  1:22  warning  ActionTree not found in 'vuex'    import/named
  1:34  warning  MutationTree not found in 'vuex'  import/named
```

When I downgrade eslint-import-resolver-node to v0.3.4, error is gone.

https://github.com/vuejs/vuex/blob/v3.6.2/types/index.d.ts
https://github.com/vuejs/vuex/blob/v3.6.2/src/index.js

If I check vuex's code both actual code and type defs export all of the types listed in the error.

**Comment by ljharb** [comment]
https://github.com/vuejs/vuex/blob/v3.6.2/src/index.js does not list GetterTree, ActionTree, or MutationTree. I see that the type definitions do - so why aren't you using `import type` for that?

**Comment by yshrsmz** [comment]
Whoops, sorry about that. I confused with mapGetters and others. `import type` solved the problem
